### PR TITLE
Added joined_at and mobile number to expense employees object

### PR DESCRIPTION
**Pull request by generalpy101**

### CODE REVIEW

File: src/components/schemas/fields.yaml [hunk 1]
```diff
@@ -1507,3 +1507,18 @@ query_params:
...
   example: state=eq.PAID
+
+joined_at:
+  type: string
+  nullable: true
+  description: |
+    Date and time when the user joined the organization.
+  example: '2020-06-11T13:14:55.201598+00:00'
+
+mobile:
```
Comment (muleyashutosh) [inline src/components/schemas/fields.yaml:1518]: we have already defined mobile field in employees 
```   mobile:
      type: string
      nullable: True
      items:
        $ref: './fields.yaml#/phone_number'
      description: |
        Mobile number of the employee
   ```
   
   can we not use the `#/phone_number` ref here?

File: src/components/schemas/fields.yaml [hunk 2]
```diff
@@ -1507,3 +1507,18 @@ query_params:
   description: | 
     This field contains the query param string similar to the query params passed in GET call for displaying data in the list view.
   example: state=eq.PAID
+
+joined_at:
```
Comment (muleyashutosh) [inline src/components/schemas/fields.yaml:1511]: we directly use timestamptz in employees, can use the same ref and just add description
```
  joined_at:
      $ref: './fields.yaml#/timestamptz_utc'

```

File: src/components/schemas/fields.yaml (hunk 1 above)
Comment (generalpy101) [reply src/components/schemas/fields.yaml:1518]: oh using it then

**Review by muleyashutosh — CHANGES_REQUESTED** [review]
minor comments, fix and re-assign

**Review by muleyashutosh — APPROVED** [review]
lgtm 🍏

File: src/components/schemas/fields.yaml [hunk 3]
```diff
@@ -1507,3 +1507,9 @@ query_params:
...
```
Comment (Code0987) [inline src/components/schemas/fields.yaml:1511]: `nullable: true`

**Pull request merged**